client/src/app/components/review-medication/review-medication.component.html

```diff
@@ -63,6 +63,18 @@ <h4 class="modal-title pull-left">Medications</h4>
                                 </button>
                                 </td>
                             </tr>
+                            <tr *ngFor="let removedMedicationComment of selectedRemovedMedicationComments">
+                                    <td id="removedSideEffectTimeStamp">
+                                        {{removedMedicationComment.timeStamp|date:'dd/MM/yyyy'}}
+                                    </td>
+                                    <td id="removedMedicationComment">
+                                        <li>
+                                            {{removedMedicationComment.commentText}}
+                                        </li>
+                                    </td>
+                                    <td id="medicationButton">
+                                    </td>
+                                </tr>
```

Comment:
This closing </tr> tag looks a little out of line with the opening tag. Please fix alignment :)